util/start_test

```diff
@@ -1142,7 +1142,7 @@ def print_chapel_environment():
     logger.write()
     logger.write("### Chapel Environment ###")
     try:
-        out = subprocess.check_output(os.path.join(util_dir, "printchplenv"))
+        out = subprocess.check_output(os.path.join(util_dir, "printchplenv"), "--all", "--no-tidy")
```

Comment:
Does anything else consume this output?

Reply:
Not that I know of. I didn't change printchplenv's with format options like --simple, figuring those were a little more likely to be used as input somewhere.

Reply:
I looked closer at that testing, and found that line 1145 in start_test needed a fix. 
```
-        out = subprocess.check_output(os.path.join(util_dir, "printchplenv"), "--all", "--no-tidy")
+        out = subprocess.check_output([os.path.join(util_dir, "printchplenv"), "--all", "--no-tidy"])
```